JS-Algorithms/challenges.js

```diff
@@ -42,6 +54,37 @@ const COUNTRY_NAMES = ["Germany", "Norway", "Island", "Japan", "Israel"];
 
 const circularArray = (index) => {
   // YOUR CODE HERE...
+
+  if (index > 0) {
+    //Total
+    const totalElement = COUNTRY_NAMES.length;
+    let newArray = [];
+
+    //Find start index
+    let index2 = 0;
+    for (x = 0; x < index; x++) {
```

Comment:
try to keep consistency in your code, in the `for` loops below you are using let